dashboard/Dockerfile

```
@@ -67,8 +67,8 @@ USER nextjs
 
 EXPOSE 3000
 
-ENV PORT 3000
+ENV PORT=3000
 
 # server.js is created by next build from the standalone output
 # https://nextjs.org/docs/pages/api-reference/next-config-js/output
-CMD HOSTNAME="0.0.0.0" node server.js
+CMD ["HOSTNAME=0.0.0.0", "node", "server.js"]
```

Thanks for the update!
Could you change it to:

```suggestion
ENV HOSTNAME="0.0.0.0"
CMD ["node", "server.js"]
```

---

Hi @CheyuWu @400Ping 
Thanks for the suggestion. I have updated the part you mentioned.